src/codewhisperer/util/userGroupUtil.ts

```diff
@@ -52,7 +52,7 @@ export class CodeWhispererUserGroupSettings {
 
     private guessUserGroup(): UserGroup {
         const randomNum = Math.random()
-        const result = randomNum <= 1 / 2 ? UserGroup.Control : UserGroup.CrossFile
+        const result = randomNum <= 1 / 2 ? UserGroup.Control : UserGroup.Utg
```

Comment:
This has a conflict with another PR of yours.

Reply:
yes, the other one is to remove cross file from the A/B
and this one it to add utg to the A/B
i know it's a bit confusing, but I do this for seperate these 2feats in 2 PRs